tests/test_constrain_current.py

```diff
@@ -192,6 +190,8 @@ def test(stellarator, grid_type):
                 )
 
         # Only makes sense to test on configurations with fixed current profiles.
-        test(HELIOTRON_vac, QuadratureGrid)
-        test(HELIOTRON_vac, ConcentricGrid)
-        test(HELIOTRON_vac, LinearGrid)
+        eq = desc.examples.get("ESTELL")
```

Comment:
When testing with heliotron_vac, we needed to add atol=1e-7 to this test (line 187) because rtol -> infinity when the expected current was zero. If we're replacing the live solve test with a saved file, so time to solve is not an issue, should we use a less trivial example for the test (i.e. nonzero current)?

Reply:
I think it's fine. `atol=1e-7` is about as strict as you can get for the current profile because `eps/mu_0 ~ 1e-8`